test/lib/notifier_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+
+class TableTest < Minitest::Test
```

Comment:
this will override the existing `TableTest` class. Let's rename it to `NotifierTest`

Reply:
eh copy and paste sin